packages/core/test/Cardano/util/subtractValueQuantities.test.ts

```diff
@@ -36,7 +36,7 @@ describe('Cardano.util.subtractValueQuantities', () => {
       ]),
       coins: 200n
     };
-    expect(subtractValueQuantities([q1, q1])).toEqual({ assets: undefined, coins: 0n });
+    expect(subtractValueQuantities([q1, q1])).toEqual({ assets: new Map(), coins: 0n });
```

Comment:
Does this mean it no longer works when assets are `undefined`?

Reply:
I does work with undefined, the change in behavior here is that it will return an empty map when the subtraction of all assets is 0 (before it would return undefined here), I can make it return undefined in this case if we want to maintain this interface

Reply:
Ah got it, it's fine